### ISSUE TITLE
[Feature Request] sv_allowlighteffects

### ISSUE THREAD

**Issue by Maxi605** [dreamstalker/rehlds#988]
Since light are set on map compilation we are left with pretty much nothing to do to change/remove this effects and some of this effects can cause low fps on low-end PCs.

My suggestion is to add a cvar that is on by default (on meaning that the light effects are allowed) to choose if we want this effects to work on the maps.

I made this suggestion on the ReGameDLL https://github.com/s1lentq/ReGameDLL_CS/issues/877 but i was told that it might be best for it to be here.

**Comment by rtxa** [comment]
I don't think you can disable something like that, lights are set on map compilation like you said, which means there's no way you can modify this from server-side (ReHLDS), if you are talking about dynamic lights, that's something you need to modify from client-side which is already supported in Half-Life custom clients but not so sure in CS 1.6 custom clients.

**Comment by dystopm** [comment]
> I don't think you can disable something like that, lights are set on map compilation like you said, which means there's no way you can modify this from server-side (ReHLDS), if you are talking about dynamic lights, that's something you need to modify from client-side which is already supported in Half-Life custom clients but not so sure in CS 1.6 custom clients.

He didn't clarify it is reffering to the dynamic light changing of some light(_spot) entities, like flickering and others, which can be edited server-side.
